content/analyst/competitive-tracker/brand-tracker.md

```diff
@@ -0,0 +1,87 @@
+---
+lastUpdated: "02/23/2023"
+title: Brand Tracker
+description: What insights can I learn from Brand Tracker?
+---
+## **What is a brand?**
+
+ First things first, a brand is the image and “personality” a company applies to its products. It is sometimes used to narrow down the different products and services within a company. It’s also possible that the brand and company could overlap.
+
+ For example, The Procter & Gamble Co. is one company with its own brand, but it also encompasses dozens of other brands such as Tide Detergent and Pepto Bismol.
+
+ Similarly, MessageSystems, Inc is one company, but it includes brands such as [eDataSource](https://www.edatasource.com/), [Boxbe](https://www.boxbe.com/), and [SparkPost](https://www.sparkpost.com/).
+
+---
+
+### Brand and Company Tracker Video
+
+ Taken from our [Competitive Tracker Training series](/analyst/general/need-help-here-is-how-to-reach-us), please see our video below for details on our Brand and Company Tracker pages:
```

Comment:
Intercom has an embedded video below this that actually does exist (https://www.youtube.com/watch?v=faIo8qR_pnQ) if we want to add that back in